spec/factories/picture.rb

```diff
@@ -0,0 +1,9 @@
+include ActionDispatch::TestProcess
+
+FactoryGirl.define do
+  factory :picture do
+    description "Funky Monkey"
+    image { fixture_file_upload(Rails.root.join('spec', 'images', 'monkey.png'), 'image/png') }
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)